browser/ui/views/frame/vertical_tab_strip_region_view.cc

```diff
@@ -403,10 +403,7 @@ void VerticalTabStripRegionView::UpdateLayout(bool in_destruction) {
 
 void VerticalTabStripRegionView::OnThemeChanged() {
   View::OnThemeChanged();
-
-  const auto background_color = GetColorProvider()->GetColor(kColorToolbar);
-  SetBackground(views::CreateSolidBackground(background_color));
```

Comment:
Removing this line seems to have caused a bug: https://github.com/brave/brave-browser/issues/27218